### ISSUE TITLE
Mousewheel events interferes with secondary scrollbar

### ISSUE THREAD

**Issue by dstromberg2** [fullcalendar/fullcalendar-scheduler#517]
Reduced test case:
https://codepen.io/anon/pen/qwBKzb

Reproduction steps:
Hover mouse in the events grid body, and use any scroll wheel action (or two finger swipe on an apple device within the body). Then click on the top scrollbar and try to drag either direction.

Result:
The headers will scroll, but the event grid body will not scroll in sync with it.

Investigation:
In ScrollJoiner.ts, for any regular scroll action, the current scrolling element is bound as the 'masterScroller' to broadcast its scroll position changes. On scrollEnd event, the masterScroller is unassigned.
However, the 'mousewheel' and related events bind the current scrolling element as the 'masterScroller', but have no action to release the assignment of masterScroller. Therefore, once a mousewheel event is registered, no other element is able to act as master.

**Comment by acerix** [comment]
Thanks for the detailed report, I can see the issue you describe.

It seems to work fine without the `margin-bottom: 0 !important` CSS though because that scrollbar is not visible.  Are you trying to have a scrollbar at the top?

**Comment by dstromberg2** [comment]
Yes, in the use case we have for it right now, there are quite a few resources displayed, so the grid's height is pretty tall. Thus, the bottom scrollbar is usually below the fold, making it non-apparent at first that it is available, and also kind of cumbersome when interacting with resources near the top of the list and needing to scroll all the way back down to access the scrollbar. Obviously not the only solution to that problem (or probably the best), but that has been the request for layout given to us.

**Comment by acerix** [comment]
I can see how it could be useful to have to the top but the scrollbar in the header is not intended to work like that.

If you don't also need one at the bottom, you could try flipping it like this:

https://stackoverflow.com/a/18998575

You could also try manually synchronising the main scrollbar anytime the top one is scrolled.

**Comment by sergey-1500469** [comment]
Hi dstromberg2, we've the same issue. Did you find a solution to solve this?)
